Rubeus/Commands/Createnetonly.cs

```diff
@@ -16,11 +16,25 @@ public void Execute(Dictionary<string, string> arguments)
             {
                 if (arguments.ContainsKey("/show"))
                 {
-                    Helpers.CreateProcessNetOnly(arguments["/program"], true);
+                    if (arguments.ContainsKey("/username"))

```

Comment:
probably cleaner to add `username`, `domain` and `password` variables at the top and set them all to null, then set them is cli arguments are given, then there's no need for these *if* statements